Samples/PushNotifications/README.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "Push Notifications Sample" 
+page_type: sample
+languages:
+- cpp
+products: 
+- windows-app-sdk
+description: "Demonstrates how to use the windows app SDK Push Notifications APIs from an unpackaged app"
+urlFragment: PushNotifications
+---
+# Push Notifications Sample
+This sample demonstrates
+1. How an app can use the WindowsAppSDK to request a channel URI
+    * The Channel URI uniquely identifies the app for a current user on a specific device, allowing other apps and servers to send notifications to this app.
+2.	How to use the WindowsAppSDK to register an app for foreground and background activation and handle the activations.
+## Requirements
+* A web API tool (like Postman or fiddler) to send notifications.
+   * The WindowsAppSDK push notifications APIs provides everything an app needs register for and receive notifications but it's expected that an external server app will be sending the notifications. The web API tool is used to emulate that external server app to send notifications.
+* An AAD RemoteIdentifier for your app.
+   * To obtain an AAD Remote identifier, follow these [instructions](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app).
```

Comment:
Update this to point to the https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart#configure-your-apps-identity-in-azure-active-directory

Reply:
done